info.yml

```diff
@@ -16,4 +16,4 @@ description: "A VFX Example Config and our default configuration."
 
 # Required minimum versions for this item to run
 requires_shotgun_version: "v6.2.0"
-requires_core_version: "v0.18.18"
+requires_core_version: "v0.18.45"
```

Comment:
Is this core version update OK? Or is it too risky?

Reply:
Check with Francis on this.

Reply:
Francis said this was OK. But that we will probably want to update to an even newer version once all the app-store has been updated with latest changes.